docs/API.md

```diff
@@ -30,10 +30,12 @@ For additional information read root [readme](../README.md)
   - [snaps methods](#snaps-methods)
     - [installSnap](#installSnap)
     - [invokeSnap](#invokeSnap)
-    - [acceptDialog](#acceptDialog)
-    - [rejectDialog](#rejectDialog)
     - [getNotificationEmitter](#getNotificationEmitter)
     - [getAllNotifications](#getAllNotifications)
+    - [dialog](#snapDialog)
+      - [positive](#acceptDialog)
+      - [negative](#rejectDialog)
+      - [type](#typeDialog)
```

Comment:
'positive' and 'negative' words sound confusing. Semantically probably better to stick with `dialog.accept` and `dialog.reject`